patterns/hero-centered-heading.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Title: Hero Centered Heading
+ * Slug: twentytwentyfive/hero-centered-heading
+ * Categories: banner, call-to-action, featured
+ * Description: A hero with a centered heading, paragraph and button.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"metadata":{"categories":["banner"],"patternName":"twentytwentyfive/hero-centered-heading","name":"Hero centered"},"align":"full","style":{"spacing":{"padding":{"right":"0","left":"0","top":"var:preset|spacing|70","bottom":"var:preset|spacing|70"},"margin":{"top":"0","bottom":"0"}},"dimensions":{"minHeight":"0vh"}},"layout":{"type":"constrained"}} -->
```

Comment:
Do we need to keep the `metadata` code, or can we remove it? `"metadata":{"categories":["banner"],"patternName":"twentytwentyfive/hero-centered-heading","name":"Hero centered"},`

Reply:
I have never seen it in a theme in production. But I am not sure if it actually makes a difference. 🤔 At least not when the pattern is **not synced has no overrides.**
Even without patternName and meta data, the user still sees the same information in the editor.

I assume this is because the pattern code was copied from the editor?
- Perhaps there is a small risk that when changes are made over time, the name and categories no longer match.

There are no themes in the theme directory that use it: https://wpdirectory.net/search/01J61B77BAHTM2MYF0N5X7BW0C

No matter what we decide,  I think it should be consistent.